apps/api/services/log_store.py

```
@@ -0,0 +1,33 @@
+            f"@{settings.tidb_host}:{settings.tidb_port}/{settings.tidb_database}"
+        )
+        _engine = create_engine(url, connect_args=connect_args, pool_pre_ping=True)
+    return _engine
+
+def ping() -> bool:
+    """DB疎通確認。ヘルスチェックに使用。"""
+    try:
+        with _get_engine().connect() as conn:
+            conn.execute(text("SELECT 1"))
+        return True
+    except Exception:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)